packages/clerk-js/src/ui.retheme/primitives/Button.tsx

```diff
@@ -58,11 +58,11 @@ const { applyVariants, filterProps } = createVariants((theme, props: OwnProps) =
         },
         neutral: {
           [vars.border]: theme.colors.$blackAlpha200,
-          [vars.accentLightest]: theme.colors.$blackAlpha50,
-          [vars.accentLighter]: theme.colors.$blackAlpha300,
-          [vars.accent]: theme.colors.$colorText,
-          [vars.accentDark]: theme.colors.$blackAlpha600,
-          [vars.accentDarker]: theme.colors.$blackAlpha700,
+          [vars.accentLightest]: theme.colors.$blackAlpha50, // TODO: once we have the new color palette and style for pseudo classes
+          [vars.accentLighter]: theme.colors.$blackAlpha300, // TODO: once we have the new color palette and style for pseudo classes
+          [vars.accent]: theme.colors.$primary800, // WIP reference: Updated to new color palette; previously `$colorText`
```

Comment:
We shouldn't have a shade of primary here, let's change it to something else for now, so it doesn't seem too out of place.